### DISCUSSION TITLE
UploadFile with requests get 422 Unprocessable Entity

### DISCUSSION THREAD

**Discussion by darrenleeleelee1** [fastapi/fastapi#6771] [Questions]
#### server
```Python
@app.post("/pdicom/{medical_number}") # save dicoms
async def post_dicom(medical_number: int, files: List[UploadFile] = File(...)):
    directory = f'./tmp/{medical_number}'
    
    
    await aiofiles.os.mkdir(directory)
    
    for file in files:
        async with aiofiles.open(directory + '/' +  file.filename, 'wb') as out_file:
            content = await file.read()  # async read
            await out_file.write(content)  # async write

    return {'Result': 'OK'}    
```
#### client
```Python
import requests

files = {'README.md': open('./README.md', 'rb')}
# header = {'Content-Type': 'multipart/form-data'}
response = requests.post('http://127.0.0.1:8000/pdicom/123', files=files)
print(response.headers)
```
### Description

* I try to use the client code to interact with the server code.
* I search a lot,and most of them say if files use dictionary type, header's Content-Type will be multipart/form-data.But I am not
![image](https://user-images.githubusercontent.com/56071442/114187063-2b830a00-997a-11eb-94ee-dfe2d18a5222.png)

* Where do I do wrong, or there has better way to interact with UploadFile data.

### Environment

* OS: Windows
* FastAPI Version: 0.61.1

* Python version: 3.8.3

**Comment by tiangolo** [comment]
I assume you solved it, thanks for closing the issue 👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
